### PR TITLE
Egységtesztek az Inventory osztályhoz

### LINKED ISSUE #5: Unit tests + Code coverage
Issue az egységteszteket és kódlefedettséget érintő feladatok dokumentálásához

### PR DESCRIPTION
**Pull request by pasztork**

### CODE REVIEW

File: src/test/vvv/InventoryTest.java [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+    @Test
+    void fillAll_OneSlot_Succeed() {
+        // Arrange
+        Slot s = getFilledSlot();
+
+        // Act
+        inventory.fillAll(s);
+
+        // Assert
+        assertEquals(ITEM_COUNT_PER_SLOT, inventory.getAminoAcidCount());
```
Comment (talmos723) [inline src/test/vvv/InventoryTest.java:36]: Please write message to assert so if it fails it tells what failed

File: src/test/vvv/InventoryTest.java [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+        assertNull(inventory.getEquipment());
+    }
+
+    @Test
+    void takeOutEquipment_NoEquipment_Fail() {
+        // Arrange
+        // Nothing to do here
+
+        // Act
+        assertThrows(IndexOutOfBoundsException.class,
```
Comment (talmos723) [inline src/test/vvv/InventoryTest.java:175]: Please move this assert to asserts from act

File: src/test/vvv/InventoryTest.java (hunk 1 above)
Comment (pasztork) [reply src/test/vvv/InventoryTest.java:36]: The environment tells where the failure happened.

File: src/test/vvv/InventoryTest.java (hunk 2 above)
Comment (pasztork) [reply src/test/vvv/InventoryTest.java:175]: The action happens inside the assertThrows method.

**Review by talmos723 — APPROVED** [review]

**Pull request merged**